src/Symfony/Component/DependencyInjection/Loader/schema/dic/services/services-1.0.xsd

```diff
@@ -204,4 +205,10 @@
       <xsd:pattern value="(%.+%|true|false)" />
     </xsd:restriction>
   </xsd:simpleType>
+
+  <xsd:simpleType name="boolean_or_star">
+    <xsd:restriction base="xsd:string">
+      <xsd:pattern value="(%.+%|true|false|\*)" />
```

Comment:
Ok got it. But we cannot change it because it was already allowed since 2.8 (because it was using the common `boolean` type that allows this usage).